### ISSUE TITLE
'tenants' query parameter for DELETE unrecognized

### ISSUE THREAD

**Issue by kreynoldsf5** [F5Networks/f5-appsvcs-extension#5]
##### ISSUE TYPE

 - Bug Report or Documentation Issue

##### AS3 BUILD/ VERSION
```
 f5-appsvcs-3.0.0-34.noarch.rpm 
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.0.6
  Build       0.0.3
  Edition     Point Release 6
  Date        Fri Apr 20 18:04:26 PDT 2018
```

##### SUMMARY

DELETE method doesn't recognize the "tenants" query parameter as described in the [documentation](http://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/3/refguide/as3-api.html#post-ref).

>DELETE https://192.0.2.10/mgmt/shared/appsvcs/declare?tenants=T1,T2,T5

>removes Tenants T1, T2, and T5 leaving the rest of the most recent declared configuration for localhost in place (assuming there are other Tenants, such as T3 and T4).

>You can also remove declarations or particular Tenants using POST instead of DELETE. You must POST a request document with action=remove and a suitable declaration. For localhost, we recommend using DELETE to remove declarations.

##### STEPS TO REPRODUCE

```
send DELETE to https://{{bigip}}/mgmt/shared/appsvcs/declare?tenants=Sample_03,Sample_04,Sample_05 where the example tenants exist.
```

##### EXPECTED RESULTS

Tenants specified in the query parameter are deleted.
##### ACTUAL RESULTS

```
{
    "code": 400,
    "message": "unrecognized URL query parameter 'tenants'"
}
```

**Comment by kaustriaf5** [comment]
Correct, the docs need to be updated. We don't support tenants as query param for any method.

**Comment by JosephPJordan** [comment]
Thanks.  Published doc revision 1.1 (http://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/3/refguide/revision-history.html) with this fix.  Closing this issue. Thanks
